docs/rl-algorithms/rpo.md

```diff
@@ -0,0 +1,129 @@
+
+## Overview
+
+RPO leverages a method of perturbing the distribution representing actions. The goal is to encourage high-entropy actions and provide a better representation of the action space. The method consists of a simple modification on top of the objective of the PPO algorithm. Assuming PPO represents action with a parameterized distribution with mean and standard deviation (e.g., Gaussian). In the RPO algorithm, the mean of the action distribution is perturbed using a random number drawn from a Uniform distribution.
+
+Original paper: 
+
+* [Robust Policy Optimization in Deep Reinforcement Learning](https://arxiv.org/abs/2212.07536)
+
+## Implemented Variants
+
+
+| Variants Implemented      | Description |
+| ----------- | ----------- |
+| :material-github: [`rpo_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/rpo_continuous_action.py), :material-file-document: [docs](/rl-algorithms/rpo/#rpo_continuous_actionpy) | For classic control tasks like Gym `Pendulum-v1`, dm_control, Pybullet. |
+
+Below are our single-file implementations of RPO:
+
+## `rpo_continuous_action.py`
+
+`rpo_continuous_action.py` works with Gym (Gymnasium), dm_control, Pybullet environments with continuous action and vector observations.
+
+The [rpo_continuous_action.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/rpo_continuous_action.py) has the following features (similar to [ppo_continuous_action.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_continuous_action.py)):
+
+* For continuous action space. Also implemented Mujoco-specific code-level optimizations
+* Works with the `Box` observation space of low-level features
+* Works with the `Box` (continuous) action space
+* adding experimental support for [Gymnasium](https://gymnasium.farama.org/)
+* 🧪 support `dm_control` environments via [Shimmy](https://github.com/Farama-Foundation/Shimmy)
+
+### Usage
+
+```bash
+# mujoco v4 environments
+poetry install --with mujoco
+python cleanrl/rpo_continuous_action.py --help
+python cleanrl/rpo_continuous_action.py --env-id Hopper-v2
+# dm_control v4 environments
+poetry install --with mujoco,dm_control
+python cleanrl/rpo_continuous_action.py --env-id dm_control/cartpole-balance-v0
+# backwards compatibility with mujoco v2 environments
+poetry install --with mujoco_py,mujoco
+python cleanrl/rpo_continuous_action.py --env-id Hopper-v2
+```
+
+### Explanation of the logged metrics
+
+See [related docs](/rl-algorithms/ppo/#explanation-of-the-logged-metrics) for `ppo.py`.
+
+### Implementation details
+Similar to PPO [ppo_continuous_action.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_continuous_action.py) with a change in `get_action_and_value` method in `Agent` class.
+
+[TODO: Add code snippets, and comparison with PPO]
+
+### Experiment results
+
+**DeepMind Control**
+
+Results on all dm_control environments. The PPO and RPO run for 8M timesteps, and results are computed over 10 random seeds.
```

Comment:
Please add a GitHub snippet embed pointing to `rpo.sh` to show the benchmark commands to help with reproducibility, like the following. Feel free to take a look at other docs as an example.

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/5555347/209712999-6636b19b-c80e-4d6f-81ae-740e3104ce45.png">